firstvoices/backend/tests/test_apis/test_languages_api.py

```diff
@@ -397,10 +397,13 @@ def test_search_mixed_sites_and_languages(self, db, mock_search_query_execute):
     def assert_language_response(self, language, language_response):
         assert language_response["language"] == language.title
         assert language_response["languageCode"] == language.language_code
+        assert language_response["id"] == str(language.id)
 
     def assert_more_sites_response(self, more_response):
-        assert more_response["language"] == "More FirstVoices Sites"
+        assert more_response["language"] == ""
```

Comment:
I guess this is related to the MoreSitesResponseSerializer which was renamed to LanguagePlaceholderSerializer. I believe we can rename this as well.